src/compas/robots/resources/basic.py

```diff
@@ -70,18 +92,49 @@ def can_load_mesh(self, url):
         is_local_file = len(scheme) in (0, 1) or scheme == 'file'
 
         if is_local_file:
-            return True
+            if os.path.isfile(url):
```

Comment:
This one breaks when the file uses the `file:///` scheme, e.g. if you try to check for 'file:///C:/Users/myuser/some_file_that_does_exist.obj' it will return `False` because it cannot deal with the scheme.

Check the `load_mesh` function, it handles that by stripping it if needed, and I guess you need to do the same thing here.

Reply:
Actually, it's great you point this out because I had overlooked to look for the `basepath` when calling `load_mesh()`  in the `DefaultLoader`. However, this makes me think if we should rather hadle the `file:///` stripping part in the `_get_local_path()` method instead of within `load_mesh()`.

Something like: 
```
    def _get_local_path(self, url):
        if url.startswith('file:///'):
            url = url[8:]
        
        basepath = self.attr.get('basepath') 
        if basepath:
            return os.path.join(basepath, url)

        return url
```

Which would imply calling this function _twice_, and changing line 92 from `is_local_file = len(scheme) in (0, 1) or scheme == 'file'` to something like: `is_local_file = len(scheme) in (0, 1)`

What do you think?

Reply:
Good point, let me check how it looks like now

Reply:
So this one we could say is resolved with the latest revisions, right?